sql/log_event.h

```diff
@@ -5283,6 +5283,61 @@ class Delete_rows_compressed_log_event : public Delete_rows_log_event
 #endif
 };
 
+/**
+  @class Wsrep_write_set_meta_log_event
+
+  Log the write set metadata for the WSREP plugin. The event is logged
+  by Galera node before XID event.
+
+  <table id="Wsrep_write_set_meta_log_event_format">
+    <caption>Wsrep_write_set_meta_log_event format</caption>
+    <tr>
+      <th>Symbol</th>
+      <th>Format</th>
+      <th>Description</th>
+    </tr>
+    <tr>
+      <td>SEQNO</td>
+      <td align="right">8</td>
+      <td>Sequence number of the write set</td>
+    </tr>
+    <tr>
+      <td>UUID</td>
+      <td align="right">16</td>
+      <td>UUID of the write set</td>
+    </tr>
+    <tr>
```

Comment:
Flags field is missing. Is it necessary to introduce at this point or can it be added later on if needed?